src/data/models/Besttime.js

```diff
@@ -1,7 +1,7 @@
 import DataType from 'sequelize';
 import Model from '../sequelize';
 
-const BestTime = Model.define(
+const Besttime = Model.define(
```

Comment:
Why lowercase it?

Reply:
It was mixed in the code so I unified it to same as Battletime.

Reply:
When in doubt we should use same notation as in SQL schema?

Reply:
Well the table names are all lowercase..

Reply:
Where they're columns they're camelcase.

Reply:
This is not a column, it's a model of a database table.